src/Akka.Persistence.Azure/Journal/AzureTableStorageJournal.cs

```diff
@@ -365,10 +370,10 @@ protected override async Task<IImmutableList<Exception>> WriteMessagesAsync(IEnu
                             if (_log.IsDebugEnabled && _settings.VerboseLogging)
                                 _log.Debug("Attempting to write batch of {0} messages to Azure storage", batchItems.Count);
 
-                            var response = await Table.SubmitTransactionAsync(batchItems, _shutdownCts.Token);
+                            var response = await Table.ExecuteBatchAsLimitedBatches(batchItems, _shutdownCts.Token);
```

Comment:
Need to explain somewhere why we call this method and what the historical reasons were, that we we don't accidentally remove this method again in the future. i.e. this is an Azure Table Storage API limitation we're working around and _we know_ this breaks atomicity for batches over 100 but we can't do anything about that.